bookwyrm_backend/src/main/java/com/bookwyrm/backend/book/dao/BookDao.java

```diff
@@ -16,7 +18,8 @@
     private String description;
     private String author;
     private String isbn;
-    private String genre;
+    private Map<String, Float> avg;
+    private Map<String, Integer> num;
```

Comment:
num could use a better name. what is it the number of? if it's ratings, then perhaps `ratingNum` would be more appropriate. avg should probably also be ratingAvg